src/api/helpers/loggingHelpers.ts

```diff
@@ -50,8 +50,21 @@ const errorLogger = winston.createLogger({
   ),
 });
 
+const infoLoggerWrapper = {
+  info: (message: string, traceId: string) => logger.info(`${message}, [traceId=${traceId}]`),
+};
+
+const errorLoggerWrapper = {
+  error: (message: string, traceId: string) =>
+    errorLogger.error(`${message}, [traceId=${traceId}]`),
+};
+
 export const getLoggers = () => {
-  return [logger, errorLogger];
+  return {
```

Comment:
Changed this to an object so that we didn't need to specify `logger`, etc. if we only needed `errorLogger`. Also so the order doesn't matter.